app/templates/components/service-link.html

```diff
@@ -1,6 +1,6 @@
-{% macro current_service_link(endpoint, link_text) %}
-  {% if current_service %}
-    <a class="govuk-link" href="{{ url_for(endpoint, service_id=current_service.id) }}">{{ link_text }}</a>
+{% macro service_link(service, endpoint, link_text) %}
+  {% if service %}
+    <a class="govuk-link" href="{{ url_for(endpoint, service_id=service.id) }}">{{ link_text }}</a>
```

Comment:
Can we add the class to remove the visited state from this?
```suggestion
    <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for(endpoint, service_id=service.id) }}">{{ link_text }}</a>
```